tracer/test/snapshots/AspNetMvc4Tests.Integrated.WithFF.__path=_Home_BadRequestWithStatusCode-statuscode=503&TransferRequest=true_statusCode=503.verified.txt

```diff
@@ -59,11 +63,15 @@
 System.Exception: Oops, it broke. Specified status code was: 503
 at Samples.AspNetMvc4.Controllers.HomeController.BadRequestWithStatusCode(Int32 statuscode),
       error.type: System.Exception,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: /error/index,
```

Comment:
I am so perplexed, this is an error that I was thinking only occurred in the `WithExpansion` scenario but I guess it also happens with the `WithFF` scenario 😵‍💫 There's a lot going on so I'm not sure if we have to solve this in this PR, but we should solve this before the next release